client/components/data/query-site-purchases/index.jsx

```diff
@@ -0,0 +1,54 @@
+/**
+ * External dependencies
+ */
+import { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { isFetchingSitePurchases } from 'state/purchases/selectors';
+import { fetchSitePurchases } from 'state/purchases/actions';
+
+class QuerySitePurchases extends Component {
+	constructor( props ) {
+		super( props );
+		this.requestSitePurchases = this.requestSitePurchases.bind( this );
```

Comment:
Do we really need to bind `requestSitePurchases()` to `this`? As far as I can see it's not used by any event handler.